test/app-luatest/gh_4963_fio_pread_err_msg_test.lua

```diff
@@ -0,0 +1,10 @@
+local fio = require('fio')
+
+local t = require('luatest')
+local g = t.group()
+
+g.test_fio_pread_err_msg = function()
+    local fh = fio.open('/tmp/', {'O_RDWR', 'O_TMPFILE'})
```

Comment:
Linux specific? I guess `/dev/null` will do the job on most Linux/Mac OS/*BSD systems.

Reply:
Good idea, thanks.